cmd/client/cmd/list.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/fatih/color"
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+)
+
+func newLsCmd() *cobra.Command {
+	lsCmd := &cobra.Command{
+		Use:   "ls",
+		Short: "List services",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			timeout, err := cmd.InheritedFlags().GetDuration("timeout")
+			if err != nil {
+				return fmt.Errorf("list services: get cli parameter 'timeout': %w", err)
+			}
+
+			ctx, cancelFunc := context.WithTimeout(context.Background(), timeout)
+			defer cancelFunc()
+
+			client, err := newClientConn(ctx, cmd)
```

Comment:
instead of creating client for every operation, how about using package level variable to declare only once ?

Reply:
A new client will be created for each command call, and the client will be set up using the provided cmd flags.